documentation/release_notes.markdown

```diff
@@ -271,17 +380,22 @@ derive data from a PuppetDB export tarball.
 
 * Allow factset hashes to be nil in db
 
-    This is necessary for factsets to function after the PDB-898 migration,
+    This is necessary for factsets to function after the ([PDB-898](https://tickets.puppetlabs.com/browse/PDB-898)) migration,
     since existing factsets are not being hashed.
 
-* Fix errors on PDB shutdown ([PDB-880](https://tickets.puppetlabs.com/browse/PDB-880))
+* Fix ActiveMQ errors on PDB shutdown ([PDB-880](https://tickets.puppetlabs.com/browse/PDB-880)) ([PDB-1102](https://tickets.puppetlabs.com/browse/PDB-1102))
+
+    We upgraded ActiveMQ to 5.7.0 (later we upgraded to 5.11.1). The
+    upgrade to 5.7.0 fixed two errors that sometimes occur at
+    shutdown. The first was related to the broker stopping before its
+    consumers were removed. The other was related to KahaDB batching
+    fails to reset due to the PageFile not being loaded. Upgrading
```

Comment:
minor in the context of everything else that's been fixed, but "The other was related to HakaDB batching fails to reset" still reads wrong